draft-barnes-acme.md

```diff
@@ -1459,14 +1450,131 @@ TODO
 
 # Security Considerations
 
-TODO
+ACME is a protocol for managing certificate that attest to identifier/key bindings.  Thus the foremost security goal of ACME is to ensure the integrity of this process.  ACME identifies clients by their account keys, so this overall goal breaks down into two more precise goals:
+
+1. Only an entity that controls a identifier can get an account key authorized for that identifier
+2. A certificate can only be managed by the holder of an account key authorized for the identifiers in the certifcate
+
+In this section, we discuss the threat model that underlies ACME and the ways that ACME achieves these security goals within that threat model.  We also discuss the denial-of-service risks that ACME servers face, and a few other miscellaneous considerations.
+
+## Threat model
+
+As a service on the Internet, ACME broadly exists within the Internet threat model {{RFC3552}}.  In analyzing ACME, it is useful to think of an ACME server interacting with other Internet hosts along three "channels":
+
+* An ACME channel, over which the ACME HTTPS requests are exchanged
+* A validation channel, over which the ACME server performs additional requests to validate a client's control of an identifier
+* A contact channel, over which the ACME server sends messages to the registered contacts for ACME clients
+
+~~~~~~~~~~
++------------+
+|    ACME    |     ACME Channel
+|   Client   |--------------------+
++------------+                    |
+       ^                          V
+       |   Contact Channel  +------------+
+       +--------------------|    ACME    |
+                            |   Server   |
+                            +------------+
++------------+                    |
+| Validation |<-------------------+
+|   Server   |  Validation Channel
++------------+
+~~~~~~~~~~
+
+In practice, the risks to these channels are not entirely separate, but they are different in most cases.  For example, a CDN employed by the ACME server might be able to tampler with a ACME messages, but not read the client's email (contact channel) or change DNS records for the client's domain names (validation channel).  
+
```

Comment:
Agree with ekr.  An email contact might be vulnerable to the various attacks that might be mounted on email, which is quite different to those that might be mounted on HTTPS.